### DISCUSSION TITLE
Timeslot system

### DISCUSSION THREAD

**Discussion by Smurfj3** [ME-MarvinE/XCalendar#153] [Q&A]
Is something like this achievable with XCalendar? 

![a07803d58a912328ceb2a2f6eb4ae69f](https://github.com/ME-MarvinE/XCalendar/assets/48727967/d1ae0e33-d2db-4d48-95e9-fe3963a49916)

**Comment by ME-MarvinE** [comment]
I have thought about making a "Scheduling" control or trying to make one using XCalendar before.

Its possible. XCalendar uses DataTemplates to show days so its hard to have information align or go across multiple of these templates which is what those lines do, and the columns in each of those days have to align with those lines.

But I think you could make this layout by visualising each column as a DayTemplate. So in each column you would have those line separators and then the blue bar under them. The hardest part would probably be the numbers on the left. You would probably need to have those in every column and then detect when a Day is in the first column in the Calendar and show/hide the numbers accordingly.
